canine/backends/remote.py

```diff
@@ -111,6 +111,9 @@ def stat(self, path: str, follow_symlinks: bool = True) -> typing.Any:
             except KeyError as e:
                 raise FileNotFoundError(path) from e
 
+    def glob(self, path: str) -> typing.List[str]:
+        raise NotImplementedError("globbing not yet implemented for remote transport!")
```

Comment:
I agree that this can come later. Whenever someone gets around to it, it should be a pretty straightforward combination of using `transport.walk()` and `fnmatch.fnmatch()`